Modules/IO/GDCM/test/itkGDCMLegacyMultiFrameTest.cxx

```diff
@@ -0,0 +1,63 @@
+/*=========================================================================
+ *
+ *  Copyright Insight Software Consortium
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0.txt
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ *=========================================================================*/
+#include <iostream>
+#include "itkGDCMImageIO.h"
+#include "itkImageFileReader.h"
+#include "itkImageFileWriter.h"
+
+// This test verifies that we obtain the correct origin and spacing for a
```

Comment:
I am not sure I understand the actual goal of this test. I do not see a comparison being done against the original (legacy) MR Image Storage Series. I guess someine validated it both are the same, but it would be good to mention that in the test.